SuperBuild.cmake

```diff
@@ -244,8 +244,8 @@ set(BRAINSTools_options
   USE_BRAINSFit:BOOL=ON
   USE_BRAINSROIAuto:BOOL=ON
   USE_BRAINSResample:BOOL=ON
-  USE_BRAINSDemonWarp:BOOL=ON
   # BRAINSTools comes with some extra tool that should not be compiled by default
+  USE_BRAINSDemonWarp:BOOL=OFF
```

Comment:
Does this mean we should set `USE_ANTS` to `ON` ?

Reply:
Good question - @hjmjohnson should we enable ANTS in Slicer's BRAINSTools build as a replacement for DemonWarp?

Reply:
Probably not.  That is a large burden for a very advanced feature that I don't think many people will appreciate.

BRAINSFit has conditional code that if ANTs exists, then one hard-coded non-linear warping method becomes available, but it is hard-coded with settings for MRI brains at 1mm voxels, and it does not have exposed parameters.  

BRAINSDemonsWarp is a mediocre non-linear warping tool that I would rather not expend energy maintaining, but it is easy and SEM compliant.  It should not take to much effort to get it compiling again.

ANTs is not SEM compliant, but it is far superior for non-linear warping.

Reply:
We could add an extension for ANTs (with a Python CLI that calls ANTs binaries) if people would like to use it in Slicer.